SharkCage/CageManager/FullWorkArea.h

```diff
@@ -0,0 +1,78 @@
+#include <Windows.h>
+#include <iostream>
+
+class FullWorkArea
+{
+public:
+	~FullWorkArea();
+	bool Init();
+private:
+	RECT rect;
+	bool GetBottomFromMonitor(int *monitor_bottom);
```

Comment:
This should also use a reference as param: `int &monitor_bottom`, changes to non-const references will be seen by the caller